dag_status.py

```diff
@@ -0,0 +1,38 @@
+import time
```

Comment:
Following [PEP 394](https://peps.python.org/pep-0394/#recommendation) this should probably be

```suggestion
#!/usr/bin/env python3

import time
```

Also I'd be happy if you could `chmod +x dag_status.py`.

Reply:
I have changed the code and linted